sopel/config/core_section.py

```diff
@@ -173,19 +173,23 @@ class CoreSection(StaticSection):
 
     :default:
         * ``NickServ`` if using the ``nickserv`` :attr:`auth_method`
+        * ``UserServ`` if using the ``userserv`` :attr:`auth_method`
         * ``PLAIN`` if using the ``sasl`` :attr:`auth_method`
 
-    The nickname of the NickServ service, or the name of the desired SASL
-    mechanism, if :attr:`auth_method` is set to one of these methods. This value
-    is otherwise ignored.
+    The nickname of the NickServ or UserServ service, or the name of the
+    desired SASL mechanism, if :attr:`auth_method` is set to one of these
+    methods. This value is otherwise ignored.
 
     See :ref:`Authentication`.
     """
 
     auth_username = ValidatedAttribute('auth_username')
     """The user/account name to use when authenticating.
 
-    May not apply, depending on :attr:`auth_method`. See :ref:`Authentication`.
+    Ignored for an :attr:`auth_method` of ``sasl``, ``server``, or
+    ``nickserv``, required for ``authserv``, ``Q``, or ``userserv``.
```

Comment:
A bit of a nitpicking here:

```suggestion
    Required by the following auth methods: ``authserv``, ``Q``, or
    ``userserv``; ignored otherwise.
```

And maybe dgw will prefer this version:

```suggestion
    Required by the following auth methods: ``authserv``, ``Q``, or
    ``userserv``—ignored otherwise.
```